tools/generate_linux_syscalls.zig

```diff
@@ -24,676 +24,240 @@ const stdlib_renames = std.StaticStringMap([]const u8).initComptime(.{
     .{ "arm_fadvise64_64", "fadvise64_64" },
```

Comment:
We should check if this list is still necessary, and if yes, whether all of the entries are.

Reply:
Checked again: All of them are necessary.